sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -38,6 +38,7 @@ import org.apache.spark.sql.types.{IntegerType, LongType, StringType, StructType
  * defined in the Catalyst module.
  */
 class SparkSqlParserSuite extends AnalysisTest {
+  import org.apache.spark.sql.catalyst.dsl.expressions._
```

Comment:
Why did you put this import here instead of the top?

Reply:
> Why did you put this import here instead of the top?

Copy from `PlanParserSuite`.....
Should I move this line to top in `PlanParserSuite` in pr  https://github.com/apache/spark/pull/29414

Reply:
Ur, I see. Its okay as it is.